doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,198 @@
+
+Build the Service VM kernel on build machine (non-root privilege)
+=================================================================
+Follow these commands to generate the Service VM kernel on your build machine::
+
+  $ cd ~
+  $ git clone http://github.com/projectacrn/acrn-kernel.git
+  $ cd acrn-kernel
+  $ git checkout acrn-2019w31.5-140000p
+  $ cp kernel_config_sos .config
+  $ make olddefconfig
+  $ make bzImage && make modules
+  $ make modules_install INSTALL_MOD_PATH=$PWD
+
+Update the Service VM kernel
+============================
+Follow the steps to enable the new Service VM kernel and its modules::
+
+  # mount /dev/sda1 /mnt
+  # scp <build machine>:~/acrn-kernel/arch/x86/boot/bzImage /mnt/EFI/org.clearlinux/
+  # sed -i "s/$(sed -n '2p' /mnt/loader/entries/acrn.conf | cut -d'/' -f4)/bzImage/" /mnt/loader/entries/acrn.conf
+  # scp -r <build machine>:~/acrn-kernel/lib/modules/* /usr/lib/modules/
+  # sync && umount /mnt
+  # reboot
+
+How to update Preempt-RT kernel
+*******************************
+In this section, we will learn to update the Preempt-RT kernel by two methods:
+
+- Rebuild from acrn-kernel branch tree.
+- Install Preempt-RT kernel from Clear Linux bundles.
+
+You could try either of them to get the Preempt-RT kernel and then copy it onto NVMe disk.
+
+How to rebuild the Preempt-RT kernel from acrn-kernel branch tree
+=================================================================
+#. Build the Preempt-RT kernel on your build machine (non-root privilege)::
+
+   $ cd ~/acrn-kernel
+   $ git reset --hard
+   $ git clean -df
+   $ git checkout acrn-2019w31.5-143000p
+   $ cp x86-64_defconfig .config
+   $ make olddefconfig
+   $ make bzImage && make modules
+   $ make modules_install INSTALL_MOD_PATH=$PWD
+
+#. Copy the Preempt-RT kernel and modules to the target platform
+   (pass-through NVME to RTVM as the rootfs dev)::
+
+   # mount /dev/nvme0n1p3 /mnt
+   # scp <build machine>:~/acrn-kernel/arch/x86/boot/bzImage /mnt/boot/bzImage.preempt-rt
+   # scp -r <build machine>:~/acrn-kernel/lib/modules/* /mnt/lib/modules/
+   # sync && umount /mnt
+
+How to install the Preempt-RT kernel from bundle
+================================================
+#. Add ``kernel-lts2018-preempt-rt`` bundle from Clear Linux (root privilege)::
+
+   # swupd bundle-add kernel-lts2018-preempt-rt
+
+#. Copy Preempt-RT module to the NVMe disk:
+
+   .. code-block:: console
+
+      # mount /dev/nvme0n1p3 /mnt
+      # ls /usr/lib/modules/
+      4.19.59-13.iot-lts2018-preempt-rt/     4.19.36-48.iot-lts2018/           4.19.36-48.iot-lts2018-sos/        5.0.14-753.native/
+      # cp -r /usr/lib/modules/4.19.59-13.iot-lts2018-preempt-rt /mnt/lib/modules/
+      # sync && umount /mnt
+
+Launch the RTVM using the following script and command
+======================================================
+.. _launch_rtvm script:
+   https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/scripts/launch_hard_rt_vm.sh
```

Comment:
The unfortunate thing here is that we already have a similar script with the **exact** same name (under devicemodel/samples/nuc). Can we consolidate those to have only one (preferred) or at least name them differently (and in a way that makes is somewhat self-explanatory)?

Reply:
Rename this script to ``launch_preempt_rtvm.sh``.
For the original one: ``devicemodel/samples/nuc/launch_hard_rt_vm.sh`` will change to launch rtvm via ``--ovmf``. Maybe we can update this section once the launch_hard_rt_vm.sh script gets ready.